requirements.txt

```diff
@@ -1 +1,2 @@
 poetry==1.5.1
+types-requests==2.31.0.6
```

Comment:
This goes in pyproject.toml, it's not required here.

Reply:
Change has done :)